src/components/elements/Accordian/Accordian.module.scss

```diff
@@ -1,22 +1,27 @@
-.LinkAccordian {
+.accordian {
   position: relative;
+  margin-bottom: 10px;
 }
 .title {
-  background: #434343;
-  padding: 10px 13px;
+  background: var(--color-grey4-grey16);
+  padding: 8px 13px;
   color: #fff;
```

Comment:
Needs to use light/dark mode color variables.

Reply:
This is fixed here b6c78755686c093e353962d26512105e1e6c5fd4  new preview links should show updates.

Reply:
Looks good